### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'compressed')

### ISSUE THREAD

**Issue by krasnyd** [jsreport/jsreport#1010]
Link to my playground: https://playground.jsreport.net/w/krasnyd/O4Fzdfqe

I am trying to merge one static-pdf with chrome-pdf template, but it returns "TypeError: Cannot read properties of undefined (reading 'compressed')". I thought of trying to compress my PDF file and it solved the issue. When you change the source of "Bowling-background" to "Bowling-compressed.pdf" it works fine, so it is a problem only with the original file "Bowling.pdf".

```
Error while executing pdf-utils operations
(because) cannot read properties of undefined (reading 'compressed')
TypeError: Cannot read properties of undefined (reading 'compressed')
    at parseObject (/app/node_modules/@jsreport/pdfjs/lib/object/reference.js:79:15)
    at PDFReference.get [as object] (/app/node_modules/@jsreport/pdfjs/lib/object/reference.js:13:17)
    at module.exports.parseBuffer (/app/node_modules/@jsreport/pdfjs/lib/parser/parser.js:101:40)
    at new ExternalDocument (/app/node_modules/@jsreport/pdfjs/lib/external.js:7:20)
    at Object.merge (/app/node_modules/@jsreport/jsreport-pdf-utils/lib/pdfManipulator.js:42:23)
    at module.exports (/app/node_modules/@jsreport/jsreport-pdf-utils/lib/pdfProcessing.js:90:27)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async AsyncFunction.<anonymous> (/app/node_modules/@jsreport/jsreport-pdf-utils/lib/worker.js:179:21)
    at async ListenerCollection.fire (/app/node_modules/@jsreport/jsreport-core/lib/shared/listenerCollection.js:157:21)
    at async afterRender (/app/node_modules/@jsreport/jsreport-core/lib/worker/render/render.js:103:5)
wrapped by:
Error: Error while executing pdf-utils operations
(because) cannot read properties of undefined (reading 'compressed')
    at module.exports (/app/node_modules/@jsreport/jsreport-core/lib/shared/createError.js:10:13)
    at WorkerReporter.createError (/app/node_modules/@jsreport/jsreport-core/lib/shared/reporter.js:49:12)
    at AsyncFunction.<anonymous> (/app/node_modules/@jsreport/jsreport-pdf-utils/lib/worker.js:196:22)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ListenerCollection.fire (/app/node_modules/@jsreport/jsreport-core/lib/shared/listenerCollection.js:157:21)
    at async afterRender (/app/node_modules/@jsreport/jsreport-core/lib/worker/render/render.js:103:5)
    at async WorkerReporter._render (/app/node_modules/@jsreport/jsreport-core/lib/worker/render/render.js:148:7)
    at async /app/node_modules/@jsreport/jsreport-core/lib/worker/reporter.js:179:19
    at async Domain.<anonymous> (/app/node_modules/@jsreport/advanced-workers/lib/workerHandler.js:141:19)
```

**Comment by pofider** [comment]
Thank you for replicating the issue for us.

The problem is that we don't support `Cross-Reference Streams` from the pdf specification chapter 7.5.8 
https://opensource.adobe.com/dc-acrobat-sdk-docs/standards/pdfstandards/pdf/PDF32000_2008.pdf

The compression did change the pdf spec to the older 1.3 which doesn't contain `Cross-Reference Streams` so that fixed the problem. 

We will try to add the support in the next versions.

Just storing here the original pdf for reference
[Bowling.pdf](https://github.com/jsreport/jsreport/files/10156402/Bowling.pdf)

**Comment by pofider** [comment]
It's fixed now, it will be available in the jsreport 3.11.0
